site/static/main.css

```diff
@@ -192,3 +192,48 @@ h6 {
     padding: 10px 10px 10px 10px;
   }
 }
+
+.float-right {
+  float: right;
+}
+
+.example {
+  display: flex;
+  flex-direction: column;
+  margin: 24px 0;
+}
+.example.side {
+  flex-direction: row-reverse;
+}
+.example .example-vis {
+  flex-grow: 1;
+  align-self: center;
+  min-width: 300px;
+  text-align: center;
+}
+.example .example-code {
+  width: 550px;
```

Comment:
Note that the width of the page is not fixed any more! Check out on small screens.